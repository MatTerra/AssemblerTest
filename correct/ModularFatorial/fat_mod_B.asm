BEGIN
SECTION DATA
	AUX: SPACE
SECTION TEXT
FAT: EXTERN
	N: EXTERN
	PUBLIC MOD_B
MOD_B:	STORE AUX
	MUL N
	STORE N
	LOAD AUX
	JMP FAT

END
