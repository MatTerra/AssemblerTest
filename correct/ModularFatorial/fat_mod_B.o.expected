H: fat_mod_B
H: 11
H: 01010101010
H: U fat 9
H: U n 3 5
H: D mod_b 0
T: 11 10 03 00 11 00 10 10 05 00 0
